cs/src/Management/TunnelManagementClient.cs

```diff
@@ -403,7 +403,8 @@ private static void ValidateHttpHandler(HttpMessageHandler httpHandler)
                         if (response.Headers.WwwAuthenticate?.Count > 0)
                         {
                             ex.SetAuthenticationSchemes(
-                                response.Headers.WwwAuthenticate.Select((v) => v.Scheme).ToArray());
+                                response.Headers.WwwAuthenticate.Select(
+                                    (v) => v.ToString()).ToArray());
```

Comment:
Relay service uses this authentication schemes of `UnauthorizedAccessException` [here](https://devdiv.visualstudio.com/OnlineServices/_git/tunnels?path=/src/Tunnel/Src/DataPlane/Relay/ControlPlaneConnection/Exceptions/ControlPlaneUnauthorizedAccessException.cs&version=GBmain&line=28&lineEnd=29&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents)
Does it need to be updated?

Reply:
It will, but not immediately. Currently the control plane only returns the simple scheme name in the `WWW-Authenticate` response header, so this `.ToString()` is the same as `.Scheme` was before.

For domain custom auth, I am changing the control plane to return additional parameters here, and updating the relay service and CLI to handle the optional additional parameters.